R/orderly.R

```diff
@@ -49,9 +54,11 @@ orderly_new <- function(name, template = NULL, force = FALSE,
                         root = NULL, locate = TRUE) {
   root <- root_open(root, locate, require_orderly = TRUE, call = environment())
   dest <- file.path(root$path, "src", name)
+  orderly_name <- deprecate_old_orderly_name(dest, name, suppress_errors = TRUE)
+  report_name <- sprintf("%s.R", name)
```

Comment:
It's a bit confusing what the two variables mean here and how they differ from one another. I think `existing_report_file` and `new_report_file` would better explain the difference.